src/www/data.njk.json

```diff
@@ -32,6 +32,9 @@
         {
           "name": "Buttons"
         },
+        {
+          "name": "Content card"
+        },
         {
           "name": "Content Tiles"
```

Comment:
```suggestion
          "name": "Content Tiles (deprecated)"
```

Reply:
This changes the content tiles page url to `.../content-tiles-(deprecated)`, is that ok?

Reply:
This also causes the sidebar links to behave strangely (it disappears and reappears lol) so I've removed (deprecated) from the page name